curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-032.md

```diff
@@ -1,32 +1,26 @@
 ---
-id: 6193f85ab1de4356da678090
+id: 61fdaea3999cb19d76ce717b
 title: Step 32
 challengeType: 0
 dashedName: step-32
 ---
 
 # --description--
 
-Now you need to align the text content of your sheet so that it appears in columns. Start with a `.row` selector to target all of your elements that have the balance values in them. Give this selector a `display` property set to `flex`, and a `justify-content` property set to `flex-end`.
+Now you need to size these elements down. Give your `span[class~="sr-only"]` selector a `width` and `height` property set to `1px`.
```

Comment:
No visible changes again and not really an explanation of why we are adding these properties. I'm fine with it - just pointing it out if you want to change it at all.